### ISSUE TITLE
Dependency package needed for TwoBit Data Manager

### ISSUE THREAD

**Issue by blankenberg** [galaxyproject/tools-devteam#94]
Currently listed as requiring 'ucsc_tools', needs to have TS installable package that provides the 'faToTwoBit' exectuable.

**Comment by bgruening** [comment]
@blankenberg what is your preferred way to do this?

* [ ] downloading latest version from UCSC mirror (I do this in deepTools)
* [ ] mirroring and versioning the UCSC binaries by ourself (we did that once in my github repository)
* [ ] build the entire ucsc src tree with https://github.com/natefoo/docker-build (we have some initial work on this)

**Comment by erasche** [comment]
+1 for the last option. In IRC @natefoo mentioned possibility of getting
output files from docker build into depot
6. apr. 2015 10.29 a.m. skrev "Björn Grüning" <notifications@github.com>:

**Comment by blankenberg** [comment]
I think the preferred order of preference would be the reverse order of the check list. With importance on keeping versioned copies, instead of just the latest.

**Comment by bgruening** [comment]
@blankenberg I think the only missing piece is to convince @natefoo to put the compiled binaries into the Galaxy depot :)

Do we want to have one `package_ucsc_2015_1` package with all binaries or do we want to have X packages for every needed binary? I think the entire UCSC source tree is released every 3 month.

**Comment by erasche** [comment]
@bgruening UCSC are versioned, they just don't keep old versions. https://github.com/natefoo/docker-build/blob/master/ucsc/build.sh#L4 We'll need to update that on a regular basis whenever builds start failing.

**Comment by bgruening** [comment]
Yes I know, imho we just need to pick on version sync it to the `depot` build from this location and we are good.

**Comment by erasche** [comment]
Ah, gotcha.

**Comment by erasche** [comment]
xref https://github.com/galaxyproject/docker-build/issues/10

**Comment by bgruening** [comment]
@blankenberg this one can be closed correct?
